test/flycheck-test.el

```diff
@@ -3053,6 +3053,13 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
    "language/erlang/rebar3/src/erlang-warning.erl" 'erlang-mode
    '(6 nil warning "wrong number of arguments in format call" :checker erlang-rebar3)))
 
+(flycheck-ert-def-checker-test erlang-rebar3 erlang build
+  (flycheck-ert-should-syntax-check
+   "language/erlang/rebar3/_checkouts/dependency/src/dependency.erl" 'erlang-mode)
+  (should (not (file-exists-p
+                (flycheck-ert-resource-filename
+                 "language/erlang/rebar3/_build/default/lib/dependency/_build")))))
```

Comment:
What's this assertion for?  What does it test?  Why is it important this this file/directory does not exist?